lib/sgov/sgov-adapter.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {SgovAdapter} from "./sgov-adapter";
+import {httpFetch} from "../io/fetch/fetch-nodejs";
+import {PimClass} from "../pim/model";
+import {CimAdapter, IriProvider, PrefixIriProvider} from "../cim";
+import {FetchOptions} from "../io/fetch/fetch-api";
+import {CoreResourceReader} from "../core";
+
+let iriProvider: IriProvider;
+let adapter: CimAdapter;
+
+beforeAll(() => {
+  iriProvider = new PrefixIriProvider();
+  adapter = new SgovAdapter("https://slovník.gov.cz/sparql", httpFetch);
+  adapter.setIriProvider(iriProvider);
+});
+
+test("SgovAdapter.search()", async () => {
```

Comment:
I know this test the search method but I would add more information about the test, like in here we search for a resource that exists. In addition there can be test that search for resources that does not exist. Or a search for a resource that has two different representations .. etc.